Scenarios Objectives -- INFO-ATHN (#605)
OWASP/wstg: document/4-Web_Application_Security_Testing/04-Authentication_Testing/05-Testing_for_Vulnerable_Remember_Password.md

- Applications provide a *remember me* functionality that allows the user to stay authenticated for long periods of time, without asking the user again for their credentials.
- Password Managers - including browser password managers - that allow the user to store their credentials in a secure manner and later on inject them in user-forms without any user intervention.

## Test Objectives

> - Validate that the generated session is managed securely and in no way put the user's credentials in danger.

Reviewer: ```suggestion
- Validate that the generated session is managed securely and do not put the user's credentials in danger.
```
